tests/blocks/block_170.cairo

```diff
@@ -1,5 +1,7 @@
 use super::state::{Block, Header, Transaction, OutPoint, TxIn, TxOut};
 
+// block containing first P2P tx to Hal Finney
```

Comment:
These comments are cool and necessary. The problem is that probably we will need to regenerate those files since data structures will be evolving. In order to prevent losing the comments I would:
1. add a `get_blocks` script along the `get_block` with all `get_block` invocations.
2. for now I would add block description in the comments or better if you can find a way to pass it to the jq script add it to the generated file itself.